src/extension/module.js

```diff
@@ -2865,6 +2907,14 @@
       // toggle sidekick
       window.hlx.sidekick.toggle();
     }
+    //The call to sampleRUM should stay in the initialization because this is run whenever sidekick
+    //is initialized in a tab; which is everytime a tab is opened, even if it is hidden. 
```

Comment:
But you aren't checking if it is shown or hidden. In case of the bookmarklet, this part is executed multiple times per tab, so you should move it to row 2906. And why only in the editor?

Also, if the goal is to track a single load event, why not call it `sidekick:load`?

Reply:
it is important that the RUM is only sampled on pages that are really enabled by the sidekick. you don't want them to send pings for other tabs. I think, it should only send a ping, if the sidekick is opened and visible on the page.